.claude/skills/docx/scripts/office/pack.py

```
@@ -0,0 +1,161 @@
+"""Pack a directory into a DOCX, PPTX, or XLSX file.
```

Ruff (0.15.7)

[warning] The file is executable but no shebang is present

(EXE002)